modules/sdk-coin-near/src/near.ts

```diff
@@ -387,7 +387,9 @@ export class Near extends BaseCoin {
       .plus(new BigNumber(transferCost.execution).plus(receiptConfig.execution).multipliedBy(gasPriceSecondBlock));
     // adding some padding to make sure the gas doesn't go below required gas by network
     const totalGasWithPadding = totalGasRequired.multipliedBy(1.5);
-    const netAmount = availableBalance.minus(totalGasWithPadding).toFixed();
+    const feeReserve = BigNumber(50000000000000000000000);
+    const storageReserve = BigNumber(2000000000000000000000); // feeReserve + storageReserve is minimum account balance for a NEAR wallet https://docs.near.org/integrator/faq#is-there-a-minimum-account-balance
```

Comment:
should we use static value in config?

Reply:
There doesn't seem to be a logical place to hold these values, since they are NEAR specific. These values are stored in a config file of bgms, but there is no way to pull from that.

Reply:
```import config from '../../../../../config/config';```
you can import from config